src/metatrain/cli/train.py

```diff
@@ -192,91 +192,81 @@ def train_model(
 
     train_size = 1.0
 
-    ###########################
-    # SETUP TEST SET ##########
-    ###########################
-
-    logger.info("Setting up test set")
-    test_datasets = []
-    if isinstance(options["test_set"], float):
-        test_size = options["test_set"]
-        train_size -= test_size
+    ############################
+    # SET UP VALIDATION SET ####
+    ############################
 
-        generator = torch.Generator()
-        if options["seed"] is not None:
-            generator.manual_seed(options["seed"])
+    logger.info("Setting up validation set")
+    val_datasets = []
+    if isinstance(options["validation_set"], float):
+        val_size = options["validation_set"]
+        train_size -= val_size
```

Comment:
Maybe this is a stupid question, but why do we subtract the val_size from the train_size here? Does it imply that we haven't provided the validation set, and thus the validation set is taken from the training set?